components/gem/actions/create-candidate/create-candidate.mjs

```diff
@@ -0,0 +1,171 @@
+  props: {
+    gem,
+    createdBy: {
+      propDefinition: [
+        gem,
+        "createdBy",
+      ],
+    },
+    firstName: {
+      type: "string",
+      label: "First Name",
+      description: "Candidate's first name",
+      optional: true,
+    },
+    lastName: {
+      type: "string",
+      label: "Last Name",
+      description: "Candidate's last name",
+      optional: true,
+    },
+    nickname: {
+      type: "string",
+      label: "Nickname",
+      description: "Candidate's nickname",
+      optional: true,
+    },
+    primaryEmail: {
+      type: "string",
+      label: "Primary Email Address",
+      description: "Candidate's primary email address",
+    },
+    additionalEmails: {
+      type: "string[]",
+      label: "Email Addresses",
+      description: "List of candidate's additional email addresses",
+      optional: true,
+    },
+    linkedInHandle: {
+      type: "string",
+      label: "LinkedIn Handle",
+      description: "If `LinkedIn Handle` is provided, candidate creation will be de-duplicated. If a candidate with the provided `LinkedIn Handle already exists, a 400 error will be returned with `errors` containing information on the existing candidate in this shape: `{\"errors\": { \"duplicate_candidate\": { \"id\": \"string\", \"linked_in_handle\": \"string\" }}}`.",
+      optional: true,
+    },
+    title: {
+      type: "string",
+      label: "Title",
+      description: "Candidate's job title",
+      optional: true,
+    },
+    company: {
+      type: "string",
+      label: "Company",
+      description: "Candidate's company name",
+      optional: true,
+    },
+    location: {
+      type: "string",
+      label: "Location",
+      description: "Candidate's location",
+      optional: true,
+    },
+    school: {
+      type: "string",
+      label: "School",
+      description: "Candidate's school",
+      optional: true,
+    },
+    educationalInfo: {
+      type: "string[]",
+      label: "Educational Info",
+      description: "A list of objects containing candidate's educational information. **Format: [{\"school\": \"string\", \"parsed_university\": \"string\", \"parsed_school\": \"string\", \"start_date\": \"string\", \"end_date\": \"string\", \"field_of_study\": \"string\", \"parsed_major_1\": \"string\", \"parsed_major_2\": \"string\", \"degree\": \"string\"}]**. [See the documentation](https://api.gem.com/v0/reference#tag/Candidates/paths/~1v0~1candidates/post) for further details.",
+      optional: true,
+    },
+    workInfo: {
+      type: "string",
+      label: "Work Info",
+      description: "A list of objects containing candidate's work information. **Format: [{\"company\": \"string\", \"title\": \"string\", \"work_start_date\": \"string\", \"work_end_date\": \"string\", \"is_current\": \"string\"}]**. [See the documentation](https://api.gem.com/v0/reference#tag/Candidates/paths/~1v0~1candidates/post) for further details.",
+      optional: true,
+    },
```

Comment:
The description says `workInfo` is a list of objects. Should the type be `string[]`?
```suggestion
    workInfo: {
      type: "string[]",
      label: "Work Info",
      description: "A list of objects containing candidate's work information. **Format: [{\"company\": \"string\", \"title\": \"string\", \"work_start_date\": \"string\", \"work_end_date\": \"string\", \"is_current\": \"string\"}]**. [See the documentation](https://api.gem.com/v0/reference#tag/Candidates/paths/~1v0~1candidates/post) for further details.",
      optional: true,
    },
```